src/user.js

```diff
@@ -0,0 +1,57 @@
+import display from './ui.js';
+
+const refreshIcon = document.querySelector('.refresh');
+const addBtn = document.querySelector('.add-btn');
+const newTask = document.querySelector('.list-input');
+const clearCom = document.querySelector('.complete-btn');
+
+const addTask = (task) => {
+  let listData = JSON.parse(localStorage.getItem('tasks')) || [];
+  if (!listData) {
+    listData = [
+      {
+        description: task,
+        completed: false,
+        index: 0,
```

Comment:
- [ ] The index should be set to the value of the new array length if the list have one task the index should be 1 not 0, kindly make sure to fix this